transform/macros/map_class_fp.sql

```diff
@@ -0,0 +1,17 @@
+{% macro map_class_fp(class_fp, k, v) -%}
+
```

Comment:
Clever, do you have a reference to link to for these codes?

Reply:
Yes! I will add it as a comment